### ISSUE TITLE
PXRDIP import not working

### ISSUE THREAD

**Issue by joelvbernier** [HEXRD/hexrdgui#636]
@bnmajor -- I was trying to use the import tool for PXRDIP, and hexrdgui hangs.  In this case, I am loading the following tiff file rather than the HDF4.  It is complaining about a zero length imageseries, then dies.  Recall that in this import case (and BBXRD will be the same) all of the detector images are in a single image file.  Image can be foud [here](https://drive.google.com/file/d/1BaGuLym_JUBxLEiVspWymyBoE9gCk_3T/view?usp=sharing)

**Comment by joelvbernier** [comment]
Also checked, and the HDF4 import isn't working either (using the archive in the same shared Google Drive folder as above).

**Comment by joelvbernier** [comment]
Update: I was reminded by @cjh1 that when running in dev mode, `pyhdf` is not installed.  That fixes the HDF4 import; I've raised a separate issue for the fabio tiff import badness in hexrd.  One thing I did notice, however, is that i have to reload the source image for every detector; any way to keep that in memory so we can go back to it without reloading?  Would prevent the user from accidentally forgetting to apply a transform...
